modules/ngx_http_concat_module/config

```diff
@@ -1,4 +1,20 @@
-ddon_name=ngx_http_concat_module
-HTTP_MODULES="$HTTP_MODULES ngx_http_concat_module"
-HTTP_AUX_FILTER_MODULES="$HTTP_AUX_FILTER_MODULES ngx_http_concat_module"
-NGX_ADDON_SRCS="$NGX_ADDON_SRCS $ngx_addon_dir/ngx_http_concat_module.c"
+ngx_addon_name=ngx_http_concat_module
+
+if test -n "$ngx_module_link"; then
+
+    ngx_module_type=HTTP_FILTER
```

Comment:
HTTP_FILTER -> HTTP_AUX_FILTER

Although HTTP_FILTER can work from auto/modules, I think HTTP_AUX_FILTER is more precise.

```
...
    if [ -z "$ngx_module_order" -a \
         \( "$ngx_module_type" = "HTTP_FILTER" \
         -o "$ngx_module_type" = "HTTP_AUX_FILTER" \) ]
    then
        eval ${ngx_module}_ORDER=\"$ngx_module_name \
                                   ngx_http_copy_filter_module\"
    else
        eval ${ngx_module}_ORDER=\"$ngx_module_order\"
    fi
...
```